docs/UserGuide.md

```diff
@@ -146,44 +146,56 @@ Examples:
 - `list` followed by `delete 2` deletes the 2nd client in the address book.
 - `find Betsy` followed by `delete 1` deletes the 1st client in the results of the `find` command.
 
+### Filter client by tag : `filter`
+Filter through clients by a specific tag provided.
+
+Format: `filter TAG_NAME`
+
+- `TAG_NAME` refers to the tag given to clients
+- Displays all clients with the specified tag
+- There should only be one tag provided else an error would be shown
+
+Examples:
+- `filter friends` Displays all clients with the tag `friends`
+
 ## Meeting Functions
 
 ### Adding a Meeting: `add`
 
 Adds a meeting for a specific client in the address book.
 
-Format: `add CLIENT_INDEX n/NAME dt/DATE_TIME v/VENUE [d/DESCRIPTION]…​`
+Format: `addMeeting clientIndex/CLIENT_INDEX dt/DATE_TIME d/DESCRIPTION`
 
 - `CLIENT_INDEX` refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​.
-- `DATE_TIME` format should be `YYYY-MM-DD HH:MM`, e.g., `2023-01-01 14:00`.
-- A meeting can have an optional description(s).
+- `DATE_TIME` format should be `YYYY-MM-DD HH:MM`, e.g., `02-01-2025 12:00`.
+- `DESCRIPTION` refers to what the meeting is about. Format should be a string.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A meeting can have any number of descriptions (including 0).
```

Comment:
Perhaps we need to update this line? Since we only have one description.

Reply:
Let me know if the changes convey what you are trying to describe